.github/workflows/master.yml

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2023 Canonical Ltd.
+
+name: Master workflow
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  build-dbtestapp:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Build
+        run: |
+          cd mongoapi/dbtestapp/
+          go build ./...
+
+  docker-build-dbtestapp:
+    env:
+      DOCKER_REGISTRY: "docker.io/"
+      DOCKER_REPOSITORY: "omecproject/"
```

Comment:
If we do `docker`-push` only for `push` actions, we also need to remove these `ENV` that I added earlier today
```suggestion
```

Reply:
Done